var/spack/repos/builtin/packages/py-picmistandard/package.py

```diff
@@ -10,16 +10,26 @@ class PyPicmistandard(PythonPackage):
     """Standard input format for Particle-In-Cell codes"""
 
     homepage = "https://picmi-standard.github.io"
-    url      = "https://github.com/picmi-standard/picmi/archive/refs/tags/0.0.14.tar.gz"
     git      = "https://github.com/picmi-standard/picmi.git"
+    pypi     = 'picmistandard/picmistandard-0.0.17.tar.gz'
+
 
     maintainers = ['ax3l', 'dpgrote', 'RemiLehe']
 
     version('develop', branch='master')
-    version('0.0.14', sha256='b7eefdae1c43119984226b2df358c86fdeef7495084e47b3575e3d07e790ba30')
+    version('0.0.18', sha256='68c208c0c54b4786e133bb13eef0dd4824998da4906285987ddee84e6d195e71')
+    # 0.15 - 0.17 have broken install logic: missing requirements.txt on pypi
+    version('0.0.16', sha256='b7eefdae1c43119984226b2df358c86fdeef7495084e47b3575e3d07e790ba30',
+            url='https://github.com/picmi-standard/picmi/archive/refs/tags/0.0.14.tar.gz')
+    version('0.0.14', sha256='8f83b25b281fc0309a0c4f75c7605afd5fa0ef4df3b3ac115069478c119bc8c3')
 
     depends_on('python@3.6:', type=('build', 'run'))
     depends_on('py-numpy', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('py-numpy@1.15:1', type=('build', 'run'))
    depends_on('py-scipy@1.5:1', type=('build', 'run'))
```
Haven't checked older versions, but this is the requirement in 0.0.18.